src/mip/coefficient_strengthening.jl

```diff
@@ -0,0 +1,176 @@
+"""
+Perform coefficient strengthening on each row and on each integer variables
+
+In particular, for the i-th constraints of the form
+aᵢ x ⩽ bᵢ and let xⱼ be a integer/binary variable
+If aᵢⱼ ⩾ d = bᵢ - Mᵢⱼ - aᵢⱼ (uⱼ - 1) > 0
+where Mᵢⱼ is maximal activity of the i-th row without considering xⱼ
+and uⱼ is upper bound of xⱼ, then
+aᵢⱼ ≔ aᵢⱼ - d
+bᵢ ≔ bᵢ - duⱼ
```

Comment:
This currently reads as an identity; I would rephrase to make clear that this is the transformation you apply.
```suggestion
and uⱼ is upper bound of xⱼ, then we transform
aᵢⱼ <- aᵢⱼ - d
bᵢ <- bᵢ - duⱼ
```